hack/e2e-suite/guestbook.sh

```diff
@@ -38,15 +38,22 @@ GUESTBOOK="${KUBE_ROOT}/examples/guestbook"
 function teardown() {
   ${KUBECTL} stop -f "${GUESTBOOK}"
   if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
-    gcloud compute forwarding-rules delete "${INSTANCE_PREFIX}-default-frontend" || true
-    gcloud compute target-pools delete "${INSTANCE_PREFIX}-default-frontend" || true
-  fi  
+    gcloud compute forwarding-rules delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+    gcloud compute target-pools delete -q "${INSTANCE_PREFIX}-default-frontend" || true
+  fi
 }
 
 prepare-e2e
 
 trap "teardown" EXIT
 
+if gcloud config list compute/region --format text | grep 'compute.region: None'; then
```

Comment:
Yes, I'm recommending a similar approach given the e2e zone, which is what we have to do in other tests as well (I think there are a few other cases). You actually don't care what the gcloud default is, FWIW - I run my e2es in a different zone (though the same region, I guess), than my default zone.